configure.ac

```diff
@@ -17,7 +17,7 @@ ERFA_NUMVER
 ## A library supports interfaces from current downto current - age
 ## Revision is the version of the current interface
 ## Follow the instructions in RELEASE.rst to change the version info
-ERFA_LIB_VERSION_INFO(2, 1, 1)
+ERFA_LIB_VERSION_INFO(2, 2, 0)
```

Comment:
I think this should actually be (3,1,2), based on the guidelines in https://github.com/liberfa/erfa/blob/master/RELEASE.rst#shared-library-version-info , right? That is, this is a backwards-compatible change in that it adds new functionality but doesn't change already-existing interfaces.